controllers/fenceagentsremediation_controller_test.go

```diff
@@ -166,7 +166,7 @@ var _ = Describe("FAR Controller", func() {
 				testVADeletion(vaName2, resourceDeletionWasTriggered)
 				testPodDeletion(testPodName, resourceDeletionWasTriggered)
 
-				By("Having Succeed condition set to true")
+				By("Having Succeeded, FenceAgentActionSucceeded conditions set to true, and Processing set to false")
```

Comment:
just a general note: I would avoid being too detailed in the these comments, and use something more generic like "Verifying correct conditions". The details are very readable in the actual test code, and in the test logs in case a test fails. That way you don't need to update the comment ever time the test changes 🙂